src/main/webapp/index.html

```diff
@@ -3,6 +3,7 @@
     <head> 
         <meta charset="UTF-8">
         <title>Dataflow Data Analyser</title>
+        <script id="configFile" type="text/javascript" src="config.js"></script>
         <link rel="stylesheet" href="style.css">
         <script src="script.js"></script>
```

Comment:
Maybe you could group all the script tags to be easier to be located.
```suggestion
        <link rel="stylesheet" href="style.css">
        <script id="configFile" type="text/javascript" src="config.js"></script>
        <script src="script.js"></script>
```

Reply:
Yes that's definitely neater!